### ISSUE TITLE
Links colored black after PR #252

### ISSUE THREAD

**Issue by wyatt8740** [uowuo/abaddon#285]
After PR #252 ( https://github.com/uowuo/abaddon/pull/252/commits/e21b678220ce6ca7712a46ee9fe674320e148bab ), hyperlinks are now black on my computer with no way to fix it in abaddon.

What is even supposed to be the way to add a link color to a GTK theme??

I would like this change reverted or for a description of how to fix it on my end.

**Comment by ouwou** [comment]
add this css rule to main.css or elsewhere. if you tell me your theme i can try to see why it failed to pick it up
```ini
*:link {
  color: #0000ff;
}
```

**Comment by wyatt8740** [comment]
My theme is a modification of an old version of someone else's. Attached. I had tried adding 

>*:link {
>  color: #0000ff;
> }

already in the GTK inspector but nothing happened there. I guess that STATE_FLAG thing is determined at startup.

It worked just now when put in main.css and restarting, though, so I guess I needed to do that.

[CDEtheme-Crimson-4-colors.zip](https://github.com/uowuo/abaddon/files/14951518/CDEtheme-Crimson-4-colors.zip)
